apps/web/server/lib/auth/logout/getServerSideProps.tsx

```diff
@@ -4,11 +4,6 @@ import { ssrInit } from "@server/lib/ssr";
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const ssr = await ssrInit(context);
-  // Deleting old cookie manually, remove this code after all existing cookies have expired
-  context.res?.setHeader(
-    "Set-Cookie",
-    "next-auth.session-token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;"
-  );
```

Comment:
this is handled in middleware:

https://github.com/calcom/cal.com/blob/14151b827f9b5186461a16344cd47d42cb6395d9/apps/web/middleware.ts#L89